backend/app/qa_engine.py

```
@@ -75,3 +84,35 @@ async def ask(self, question: str) -> Dict[str, Any]:
+
+        payload = {
+            "model": model,
+            "prompt": prompt,
+            "stream": False,
+        }
+        try:
+            resp = requests.post(f"{base_url.rstrip('/')}/api/generate", json=payload, timeout=20)
+            if resp.ok:
+                return resp.json().get("response", "").strip()
+        except Exception:
+            pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)